elements/pl_file_preview/pl_file_preview.css

```diff
@@ -0,0 +1,56 @@
+[id^=file-preview] .file-status-container {
```

Comment:
I think I would generally prefer to wrap a `<div class="pl_file_preview">` around the file preview and use the class as a CSS selector rather than `id`. But I don't have particularly strong opinions about CSS.